protocol/builder/record_batch_builder.go

```diff
@@ -0,0 +1,97 @@
+package builder
+
+import (
+	"time"
+
+	kafkaapi "github.com/codecrafters-io/kafka-tester/protocol/api"
+)
+
+type RecordBatchBuilder struct {
+	baseOffset           int64
+	partitionLeaderEpoch int32
+	attributes           int16
+	firstTimestamp       int64
```

Comment:
```suggestion
	baseTimestamp       int64
```

Consistency with docs